lib/pdk/validators/ruby/rubocop.rb

```diff
@@ -11,13 +11,18 @@ def self.name
       end
 
       def self.cmd
-        'pwd'
+        'rubocop'
```

Comment:
I think for rubocop, we'll need to default to the `#{PDK::Util.module_root}/lib` directory instead, because rubocop isn't liking parsing of `.rubocop.yml` file in the module_root.